### PR TITLE
refactor(dashboard): rename 'duplicate' to 'clone'

### PR DESCRIPTION
**Pull request by minseolee**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] `Error / Warning / Lint / Type`

### Description

https://cloudone-mz.slack.com/archives/C044D55N2V9/p1670830618334959

1. Contrary to cost-dashboard, duplicating dashboard feature would be executed by 'clone' button. (At cost-dashboard, duplicating feature has been provided by duplicate context item)
2. So I've applied it to codes and refactored names

### CODE REVIEW

File: src/services/dashboards/dashboard-detail/modules/DashboardMoreMenu.vue [hunk 1]
```diff
@@ -70,16 +57,16 @@ const state = reactive({
     moreMenuItems: computed(() => {
         const menuItems = cloneDeep(defaultMenuItems.value);
         if (state.homeDashboardId === props.dashboardId) {
+            menuItems[0].disabled = true;
             menuItems[1].disabled = true;
```
Comment (yuda110) [inline src/services/dashboards/dashboard-detail/modules/DashboardMoreMenu.vue:61]: I think this way is safer.
```suggestion
            menuItems.find(d => d.name === MENU.DELETE).disabled = true;
            menuItems.find(d => d.name === MENU.SET_HOME).disabled = true;
```

**Review by yuda110 — APPROVED** [review]
LGTM! I left a comment!!

File: src/services/cost-explorer/cost-dashboard/modules/CostDashboardMoreMenu.vue [hunk 2]
```diff
@@ -80,16 +80,16 @@ export default {
             moreMenuItems: computed(() => {
                 const menuItems = cloneDeep(defaultMenuItems.value);
                 if (state.homeDashboardId === props.dashboardId) {
-                    menuItems[1].disabled = true;
-                    menuItems[2].disabled = true;
+                    menuItems.find((d) => d.name === MENU.DELETE)!.disabled = true;
+                    menuItems.find((d) => d.name === MENU.SET_HOME)!.disabled = true;
```
Comment (WANZARGEN) [inline src/services/cost-explorer/cost-dashboard/modules/CostDashboardMoreMenu.vue:84]:
```suggestion
                    // can not be undefined
                    menuItems.find((d) => d.name === MENU.DELETE)!.disabled = true;
                    menuItems.find((d) => d.name === MENU.SET_HOME)!.disabled = true;
```

**Review by WANZARGEN — APPROVED** [review]
LGTM~!

**Pull request merged**